osis-core/src/main/java/com/scality/osis/utils/ScalityModelConverter.java

```diff
@@ -92,6 +95,20 @@ public static ListAccountsRequestDTO toScalityListAccountsRequest(long limit) {
                 .build();
     }
 
+    /**
+     * Created Vault List Accounts request for List Tenants
```

Comment:
Should this be ListUsers request?

Reply:
fixed in 5385bbc9255f242f654930813e49a22528eae0d9